chanzuckerberg/idseq-web
# [Data Discovery] UI fixes: main fix is to tighten up upper part of samples view.

# Description

* Samples page:
	* Reduce icon sizes and spacing between them
	* Reduce spacing between table and toolbar
	* Fix bug where disabled icons were still clickable

* General discovery view:
	* Reduce left/right side padding

<img width="1680" alt="Screen Shot 2019-04-10 at 3 30 33 PM" src="https://user-images.githubusercontent.com/37312125/55921511-fc323a80-5bb1-11e9-9849-55dacbf7336b.png">

## Testing

* Check discovery view pages for UI glitches
* On samples view, 
	* test that heatmap and save icons are only enabled when at least two samples are selected.
	* make sure these icons are not clickable when greyed out, and that they are clickable when they are blue (active)
